csrc/id_model/indexing_traversal.cpp

```diff
@@ -44,6 +44,27 @@ IndexingTraversal::IndexingTraversal(
     }
     resize_paths_.insert(resize);
   }
+
+  // A unique expr path should be always allowed
+  for (const auto& expr_g : graph.disjointExprSets().disjointSets()) {
+    auto resize = dynamic_cast<Resize*>(expr_g->front());
+    if (resize == nullptr) {
+      continue;
+    }
+
+    auto input_groups = graph.inputGroups(expr_g);
+    auto output_groups = graph.outputGroups(expr_g);
+    if (input_groups.size() != 1 || output_groups.size() != 1) {
```

Comment:
At first I thought this should be an `NVF_ERROR` but I guess this handles the case where there a `Resize` output is mapped to an ID that was another Resize, or a Split/Merge from a reshape. Is that correct?

Reply:
I actually don't remember why I chose to do this. `NVF_ERROR` seems to make more sense. Let me check.

Reply:
Assertion seems fine. Changed.